osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -128,10 +135,39 @@ private void load()
                                                             RelativeSizeAxes = Axes.X,
                                                             AutoSizeAxes = Axes.Y,
                                                             Padding = new MarginPadding { Horizontal = 5 },
-                                                            Children = new Drawable[]
+                                                            Spacing = new Vector2(0, 10),
+                                                            Children = new[]
```

Comment:
The contents here jump around a lot - part of it is because the mod display is in between the header and the button, but another part of it is because the parenting container is `Centre`-anchored; would probably make it `TopCentre` for a little less jumping around.